src/main/resources/application.properties

```diff
@@ -1,3 +1,3 @@
-server.port=8083
+server.port=8084
```

Comment:
Please change port number, other service is running on 8084 already. better change to  8085.